README.md

```diff
@@ -2,19 +2,51 @@
 
 High performance JSON serialization and deserialization library for JavaScript, Node.js, browser.
 
-`json-pack` contains implementations of various JSON codecs into binary,
-formats, such as MessagePack, CBOR and other formats.
+`json-pack` contains implementations of various JSON codecs into binary formats, such as MessagePack, CBOR and other formats.
 
-- [__MessagePack__](./src/msgpack/README.md)
-- [__CBOR__](./src/cbor/README.md)
-- DAG-CBOR
-- JSON
-- DAG-JSON
-- UBJSON
-- Amazon Ion
-- RESP3
-- Bencode
+## Supported Formats
 
+This library implements the following serialization formats:
+
+- **[MessagePack](./docs/msgpack.md)** - Fast and lean implementation of MessagePack codec
+- **[CBOR](./docs/cbor.md)** - Concise Binary Object Representation codec
+- **[UBJSON](./docs/ubjson.md)** - Universal Binary JSON codec  
+- **[JSON](./docs/json.md)** - Enhanced JSON encoder/decoder with additional features
+- **[JSON Binary](./docs/json-binary.md)** - JSON with binary data support using Uint8Array
+- **[Amazon Ion](./docs/ion.md)** - Amazon's Ion data serialization format
+- **[BSON](./docs/bson.md)** - Binary JSON format used by MongoDB
+- **[RESP](./docs/resp.md)** - Redis Serialization Protocol (v2 and v3)
+- **[Bencode](./docs/bencode.md)** - BitTorrent's encoding format
+
+Each format comes with optimized encoders and decoders designed for maximum performance.
+
+## Installation
+
+```bash
+npm install @jsonjoy.com/json-pack
+```
+
+## Quick Start
+
+```ts
+import {MessagePackEncoder, MessagePackDecoder} from '@jsonjoy.com/json-pack/lib/msgpack';
+
+const encoder = new MessagePackEncoder();
+const decoder = new MessagePackDecoder();
+
+const data = {hello: 'world', numbers: [1, 2, 3]};
+const binary = encoder.encode(data);
+const restored = decoder.decode(binary);
+
+console.log(restored); // {hello: 'world', numbers: [1, 2, 3]}
+```
+
+## Documentation
+
+For detailed documentation on each codec, see the [docs/](./docs/) folder:
+
+- [Complete documentation index](./docs/README.md)
+- Individual codec documentation linked above in the "Supported Formats" section
 
 ## Benchmarks
```

Comment:
- Move benchmarks section into a separate file under `docs/`. Link to it from main readme.
- Also link to this: "https://jsonjoy.com/blog/json-codec-benchmarks" benchmarking blog post with title "Benchmarking JSON Serialization Codecs". It measures that `json-pack` library is the fastest library in NPM ecosystem.